boards/riscv/hpm6750evkmini/hpm6750evkmini_defconfig

```diff
@@ -0,0 +1,25 @@
+CONFIG_SOC_SERIES_HPMICRO=y
+CONFIG_SOC_HPM6750=y
+CONFIG_BOARD_HPM_6750EVKMINI=y
+CONFIG_XIP=n
+CONFIG_PLIC=y
+CONFIG_RISCV_MACHINE_TIMER=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_PINCTRL=y
+
+# Serial driver options
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# GPIO driver options
+CONFIG_GPIO=y
+
+# HW DSP options
+#CONFIG_SOC_ANDES_V5_HWDSP=y
+
+# Cache options
+# CONFIG_CACHE_ENABLE=y
```

Comment:
Ditto

Reply:
I will remove it, flash mode is not currently supported and cache is not working for now. flash support will be committed after the initial release and cache will be turned on at that time.